### DISCUSSION TITLE
barras de rolagens - Po-Grid

### DISCUSSION THREAD

**Discussion by jpinto10** [po-ui/po-angular#1969] [Tire suas dúvidas]
Bom dia
Como faço para habilitar as barras de rolagens horizontal e vertical(preferencialmente), no po-grid?
Obs.: Pode manter o cabeçalho fixo!?

**Comment by jpinto10** [comment]
Seria esse estilo, mantendo o cabec fixo e a barra de rolagem

![image](https://github.com/po-ui/po-angular/assets/29482636/22677c55-5159-4581-891d-8c0f58988a39)

**Comment by anliben** [comment]
Boa Tarde @jpinto10, tudo bem?

Para podermos apoiá-lo precisaremos de mais detalhes para entender a situação

- Utilize o Stackblitz para reproduzir o problema: https://stackblitz.com/edit/po-ui

### Detalhe passo a passo a reprodução da ocorrência, inclua evidências utilizando imagens, vídeos ou gifs:

- Passo 1...*
- Passo 2...
- Detalhe o comportamento atual:
- Detalhe o comportamento que você espera ver:

### Informe a versão dos ambientes:
- PO UI: Ex. v15.8.0
- Angular: Ex. 15
- Node: Ex. 15

### Informe o Browser:
- [ ] Chrome
- [ ]  Firefox
- [ ] Edge
- [ ]  Safari
 - [ ] IE 11
- [ ] Outros

### Informe e o Sistema Operacional:
- [ ] Windows
- [ ] macOS
- [ ] Ubuntu
- [ ] Outros

### Aguardamos seu retorno!

**Reply by anliben** [reply]
Boa Tarde @jpinto10, tudo bem?

Preciso do ambiente para saber como montou seu `po-grid` conforme a forma montada pode ser seu ambiente ou se for o `po-ui` eu consigo lhe dar outra solução!

Attr.

**Comment by jpinto10** [comment]
Obs.: ja ha imagem de referencia, na segunda interação mimha, com o comportamento atual.
Fiz um video tentando mostraro processo.

Detalhe passo a passo a reprodução da ocorrência, inclua evidências utilizando imagens, vídeos ou gifs:
	Filmagem em anexo
Informe a versão dos ambientes:
	@angular-devkit/architect       0.1502.10
	@angular-devkit/build-angular   15.2.10
	@angular-devkit/core            14.2.13
	@angular-devkit/schematics      14.2.13
	@angular/cdk                    14.2.7
	@angular/material               14.2.7
	@schematics/angular             14.2.13
	rxjs                            7.5.7
	typescript                      4.9.5
	node							v18.12.1
Informe o Browser:
	•	[x] Chrome
Informe e o Sistema Operacional:
	•	[x] Windows

Seguem informações

https://github.com/po-ui/po-angular/assets/29482636/18c0b024-e165-4de9-8e51-cb40a1e1ba5b

**Comment by jpinto10** [comment]
Boa tarde.

Como faço para vc ter essa informação?!

 

De: João Victor ***@***.***> 
Enviada em: quarta-feira, 28 de fevereiro de 2024 14:59
Para: po-ui/po-angular ***@***.***>
Cc: Edilson ***@***.***>; Mention ***@***.***>
Assunto: Re: [po-ui/po-angular] barras de rolagens - Po-Grid (Discussion #1969)

 

Boa Tarde @jpinto10 <https://github.com/jpinto10> , tudo bem?

Preciso do ambiente para saber como montou seu po-grid conforme a forma montada pode ser seu ambiente ou se for o po-ui eu consigo lhe dar outra solução!

Attr.
